Isosurfacing_3/include/CGAL/Cell_type.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+
+#ifndef CGAL_DOMAIN_CELL_TYPE
+#define CGAL_DOMAIN_CELL_TYPE
+
+#include <limits>
+
+namespace CGAL {
+namespace Isosurfacing {
+
+typedef std::size_t Cell_type;
+
+static constexpr Cell_type ANY_CELL = std::numeric_limits<Cell_type>::max();
```

Comment:
Due to a problem with VC++ this must become  `(std::numeric_limits<Cell_type>::max)()`.